web/pgadmin/tools/sqleditor/__init__.py

```
@@ -2182,6 +2187,97 @@ def start_query_download_tool(trans_id):
+            status=410,
+            success=0,
+            errormsg=gettext(
+                "Could not find the required parameters (rowpos, colpos)."
+            )
+        )
+
+    try:
+        row_pos = int(data['rowpos'])
+        col_pos = int(data['colpos'])
+        if row_pos < 0 or col_pos < 0:
+            raise ValueError
```

Ruff (0.15.4)

[warning] Abstract `raise` to an inner function

(TRY301)